packages/flutter/lib/src/cupertino/magnifier.dart

```diff
@@ -9,7 +9,7 @@ import 'package:flutter/widgets.dart';
 /// finger may be blocking the point of interest, like a selection handle.
 ///
 /// Delegates styling to [CupertinoMagnifier] with its position depending on
-/// [magnifierOverlayInfoBearer].
+/// [magnifierInfoBearer].
```

Comment:
I'm on board with simplifying this.  I think it even makes sense to remove "Bearer" as you mentioned.  I don't think that's a term that's used elsewhere in Flutter, and it doesn't add much here.